docs/designers-developers/developers/themes/theme-json.md

```diff
@@ -158,66 +166,108 @@ The list of properties that are currently exposed via this method are:
 
 ### Features
 
-This is being implemented, so it's not currently available.
+So far, this function is only enabled for the `global` section in `experimental-theme.json`.
+
+```
+{
+  "global": {
+    "features": {
+      "typography": {
+        "dropCap": false
+			}
```

Comment:
There's something weird about these spaces in most of the examples. Is your editor using tabs by any chance?

Reply:
I can see it now :)

I will fix it.